### PR TITLE
[recipes.md] initial edits to recipes guide

### PR DESCRIPTION
**Pull request by sarah11918**
#### What kind of changes does this PR include?

- New or updated content

#### Description
Restructuing Recipes.md to go in order from most-likely to least-likely need for info. Also adding some nuance esp. surrounding community recipes (most common!) and editing/PRing existing recipes to update/correct.

WIP, will want @Jutanium to contribute!

### CODE REVIEW

**Review by Jutanium — COMMENTED** [review]
NWTWWHB!

File: RECIPES.md [hunk 1]
```diff
@@ -52,30 +83,15 @@ The CMS, Deploy, and Migrate guides all have sections for external links. You ca
 
 ## More Recipes page
```
[the remark this answers is not in the document]
Comment (sarah11918) [reply RECIPES.md:89]: Ooh, that was probably a heading level typo. Not mean to be inside existing. Will work on restructuring anyway.

**Review by Jutanium — APPROVED** [review]
I'm happy with this for now!

File: RECIPES.md [hunk 2]
```diff
@@ -38,44 +74,21 @@ These require:
...
-
-### Community Recipes
-
-You can share a published recipe in our “Community Recipes” section. This includes posts on a personal or company blog, or on your preferred blogging platform like [DEV.to](https://dev.to/). This follows the IndieWeb principle of “Publish (on your) Own Site, Syndicate Elsewhere” (often abbreviated to [POSSE](https://indieweb.org/POSSE)).
-
-After you publish your article, [open a PR adding a link to this page](https://github.com/withastro/docs/edit/main/src/content/docs/en/recipes.mdx).
+## Add a new official recipe
 
-### Official Recipes
+The Official Recipes section is for officially maintained recipes, found in the `src/content/guides/recipes` folder, that do not fit into one of our pre-existing categories. Like the rest of our docs, these must meet the standards of our [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) and require long-term maintenance to make sure they're up to date.
```
Comment (Yan-Thomas) [inline RECIPES.md:91]: Another singular/plural disagreement?
```suggestion
The Official Recipes section is for officially maintained recipes, found in the `src/content/guides/recipes` folder, that does not fit into one of our pre-existing categories. Like the rest of our docs, these must meet the standards of our [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) and require long-term maintenance to make sure they're up to date.
```

Comment (sarah11918) [reply RECIPES.md:91]: I think I'm actually correct here! (For a change... :sweat_smile:)

Check me:
the SECTION (sing) is for RECIPES (pl) that DO (pl) not fit....

Comment (Yan-Thomas) [reply RECIPES.md:91]: You're right, disregard this one and merge! 😜

**Pull request merged**